darwin/utils.py

```diff
@@ -84,41 +84,39 @@ def prompt(msg: str, default: Optional[str] = None) -> str:
 
 
 def find_files(
-    files: List[Union[str, Path]] = [], recursive: bool = True, files_to_exclude: List[Union[str, Path]] = []
+    files: List[Union[str, Path]], *, files_to_exclude: List[Union[str, Path]] = [], recursive: bool = True
```

Comment:
Wont this break backwards compatibility with previous clients now that we are forcing some of the arguments to be named?

Reply:
Yes, but I'm like 99.9% positive this function is not called explicitly by anybody, and even if it is, it's a helper function, not really meant to be front facing. Let's see if @simedw and @myFree agree